holoviews/core/spaces.py

```diff
@@ -495,6 +495,13 @@ def __call__(self, *args, **kwargs):
         return ret
 
 
+class OverlayCallable(Callable):
+    """
+    A Callable subclass specifically meant to indicate that the Callable
+    represents an overlay operation between two objects.
+    """
```

Comment:
I'm not convinced it is worth introducing a subclass just for isinstance checks. Why not some parameter/attribute on ``Callable``?

This is another classname to know about and it isn't clear from this docstring why callables that involve overlay operations should be special.